source/includes/5.0-fact-currentop.rst

```diff
@@ -0,0 +1,9 @@
+Starting in MongoDB 5.0, the :pipeline:`$currentOp` aggregation 
+stage is used when running the helper method :method:`db.currentOp()` 
+via :binary:`~bin.mongosh`.
+
+Given this, in the 5.0 version of the shell and via mongosh,
```

Comment:
@ianf-mongodb would you mind changing via to another word?
https://docs.mongodb.com/meta/style-guide/terminology/general-term-guidelines/avoid-obscure-words/